app/controllers/state_file/questions/az_retirement_income_subtraction_controller.rb

```diff
@@ -1,6 +1,17 @@
 module StateFile
   module Questions
     class AzRetirementIncomeSubtractionController < RetirementIncomeSubtractionController
+
+      def self.show?(intake)
+        Flipper.enabled?(:show_retirement_ui) && intake.eligible_1099rs.present?
+      end
+
+      def eligible_1099rs
+        @eligible_1099rs ||= current_intake.state_file1099_rs.select do |form1099r|
+          form1099r.taxable_amount&.to_f&.positive?
+        end
+      end
```

Comment:
i think we can remove the `eligible_1099rs` method here?

Reply:
def